provision/aws/rosa_create_cluster.sh

```diff
@@ -56,14 +56,18 @@ fi
 
 mkdir -p "logs/${CLUSTER_NAME}"
 
+function custom_date() {
+    echo "$(date -uIs | sed -e "s/:/-/g")"
```

Comment:
You might be looking for something like this:

```suggestion
    echo "$(date '+%Y%m%d-%H%M%S')"
```

Same below for the other date.

Reply:
Thanks, done